src/shared/components/FormField/FormField.tsx

```diff
@@ -1,17 +1,33 @@
 import React from 'react'
 
-import { ChildrenWrapper, FormFieldDescription, FormFieldTitle, FormFieldWrapper } from './FormField.style'
+import {
+  ChildrenWrapper,
+  FormFieldDescription,
+  FormFieldHeader,
+  FormFieldTitle,
+  FormFieldWrapper,
+} from './FormField.style'
+
+import { Text } from '../Text'
 
 export type FormFieldProps = {
   title: string
+  optional?: boolean
```

Comment:
Maybe `isOptional` would be more readable? WDYT?

Reply:
I don't know,  I kinda like `optional`. This is similar to passing `required` or `disabled` to input.